MAUI/Popup/working-with-mvvm.md

```diff
@@ -0,0 +1,444 @@
+    public ICommand OpenPopupCommand { get; set; }
+
+    public bool IsOpen
+    {
+        get
+        {
+            return isOpen;
+        }
+        set
+        {
+            isOpen = value;
+            RaisePropertyChanged("IsOpen");
+        }
+    }
+
+    public ViewModel()
+    {
+        OpenPopupCommand = new Command(OnOpenPopup);
+    }
+
+    private void OnOpenPopup()
+    {
+        IsOpen = true;
+    }
+
+    #region INotifyPropertyChanged
+
+    public event PropertyChangedEventHandler PropertyChanged;
+
+    private void RaisePropertyChanged(string propertyName)
+    {
+        if (this.PropertyChanged != null)
+        {
+            this.PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
+        }
+    }
+
+    #endregion
+}
+{% endhighlight %}
+{% endtabs %}
+
+## How to use commands for popup buttons?
+
+`SfPopup` provides command support for the popup buttons,
+
+ * [SfPopup.AcceptCommand](https://help.syncfusion.com/maui/popup/popup-events#accept-command)
+
+ * [SfPopup.DeclineCommand](https://help.syncfusion.com/maui/popup/popup-events#decline-command)
+
+## How to use MAUI popup as a page?
+
+To use `SfPopup` as a page, create a new XAML file with `SfPopup` in it. You can use this XAML file instance (SfPopup view), across any file in your project wherever you may want to display a popup, thus allowing you to reuse the same instance of the popup where you can modify or change the required properties alone based on the current context to be displayed.
+
+The below code snippet shows how to create a popup in XAML file. We have created a simple popup to show error message popup to user on a unsuccessful login.
+
+{% tabs %}
+{% highlight xaml tabtitle="PopupPage.xaml" %}
+<sfPopup:SfPopup xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                 xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                 x:Class="PopupMVVM.PopupPage"
+                 xmlns:sfPopup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+                 x:Name="sfPopup">
+    <sfPopup:SfPopup.HeaderTemplate>
+        <DataTemplate>
+            <StackLayout>
+                <Label Text="Login failed" HorizontalTextAlignment="Center"/>
+            </StackLayout>
+        </DataTemplate>
+    </sfPopup:SfPopup.HeaderTemplate>
+    <sfPopup:SfPopup.ContentTemplate>
+        <DataTemplate>
+            <StackLayout>
+                <Label Text="Incorrect credentials" VerticalOptions="Center"/>
+            </StackLayout>
+        </DataTemplate>
+    </sfPopup:SfPopup.ContentTemplate>
+</sfPopup:SfPopup>
+{% endhighlight %}
+{% highlight c# tabtitle="PopupPage.xaml.cs" %}
+using Syncfusion.Maui.Popup;
+
+public partial class PopupPage : SfPopup
+{
+    public PopupPage()
+    {
+        InitializeComponent();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+The below code snippet shows how to consume the previously created SfPopup page from inside a different content page.

```

Comment:
The following code sample shows how to consume the previously created SfPopup page from inside a different content page.